sway/config/bar.c

```diff
@@ -1,5 +1,5 @@
 #define _POSIX_C_SOURCE 200809L
-#define _XOPEN_SOURCE 700
+#define _POSIX_C_SOURCE 200809L
```

Comment:
```_POSIX_C_SOURCE``` is defined twice here.

Reply:
Good catch. Since it's defined twice with the same value the compiler didn't complain. Fixed!